### ISSUE TITLE
This project is unmaintained: Interested in forking?

### ISSUE THREAD

**Issue by fkelbert** [jpswalsh/academicons#48]
Is there any interest of forking (if allowed) and maintaining a new font providing more icons?

**Comment by musoke** [comment]
The licences certainly would allow a fork... do you mean endorsed by @jpswalsh and/or the users?

**Comment by jpswalsh** [comment]
@musoke is right—you're welcome to do whatever you want with the project!

Regarding the lack of maintenance, I'll aim to do the next update within the week. I've neglected this project for a while, and that's largely because the majority of requests don't provide what's needed (or what's wanted doesn't actually exist). However, looking at the list now I see that there are a few that have provided everything I need, and there seems to be a demand for some of them.

So, maybe wait another week and see where we stand.

**Comment by musoke** [comment]
Just some ideas that might make this run more smoothly for you in the future...

 - add [issue templates](https://github.com/blog/2111-issue-and-pull-request-templates) so that you have a better chance of getting the info needed
 - Instructions for how to add new symbols. I would have been quite happy to submit a pull request for the icon I requested, but didn't know where to start.

**Comment by fkelbert** [comment]
@musoke  I agree. I was also filing requesting for additional icons, albeit without having an idea which information was needed to easily implement the icon.

So information like "I need a two-colored vector picture of the icon" would be great.

**Comment by jpswalsh** [comment]
Thanks to both of you for the feedback. I decided to add instructions to the README file that provide some guidance on requests.

**Comment by fkelbert** [comment]
Thanks a lot, James, for resuming the work on this icon set!
